### PR TITLE
Auth-challenge for review

### PR DESCRIPTION
**Pull request by BAZBOT123**

### CODE REVIEW

File: src/server/controllers/user.js [hunk 1]
```diff
@@ -39,12 +39,14 @@ const login = async (req, res) => {
...
 };
 
 module.exports = {
     register,
     login
-};
\ No newline at end of file
+};
+
+
```
Comment (PointWolfAnj) [inline src/server/controllers/user.js:52]: good job!

Comment (nicopicchio) [inline src/server/controllers/user.js:52]: wow you've completed it with way less code than I have. and way quicker!

File: src/server/controllers/movie.js [hunk 2]
```diff
@@ -11,21 +11,35 @@ const getAllMovies = async (req, res) => {
 };
 
 const createMovie = async (req, res) => {
+    console.log('stage one:')
```
[the remark this answers is not in the document]
Comment (BAZBOT123) [reply src/server/controllers/movie.js:14]: it was just for visibility when putting my code. Just to see what what exactly going on.

File: src/server/controllers/movie.js [hunk 3]
```diff
@@ -11,21 +11,35 @@ const getAllMovies = async (req, res) => {
...
         // todo verify the token
+        console.log('hello', token)
     } catch (e) {
         return res.status(401).json({ error: 'Invalid token provided.' })
     }
 
-    const createdMovie = null;
+    const createdMovie = await prisma.movie.create({
+        data: { title, description, runtimeMins }
+    });
```
[the remark this answers is not in the document]
Comment (BAZBOT123) [reply src/server/controllers/movie.js:37]: Wasn't needed buddy.

**Pull request closed without merging**